src/main/scala/org/labrad/manager/Manager.scala

```diff
@@ -66,10 +66,14 @@ class CentralNode(
   // TODO: this should be configurable, for example passwords per host
   val externalConfig = ServerConfig(host = "", port = 0, credential = Password("", password))
 
+  // ExecutionContext for potentially long-running server work (auth server and registry)
+  lazy val serversExecutionContext =
```

Comment:
We only need this execution context if running a registry or auth server in-process. A lazy val is a simple way to ensure that we only create it if needed.